### PR TITLE
Properly support use from Meteor packages

### PR DESCRIPTION
**Pull request by graemian**
This is a fix for https://github.com/epeli/underscore.string/issues/419

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Official version [underscorestring:underscore.string](https://atmospherejs.com/underscorestring/underscore.string) doesn't work from packages - see https://github.com/epeli/underscore.string/issues/419
```
Comment (stoeffel) [inline README.markdown]: Honestly, I don't like having this on the first line in the readme. Underscore.string's primary target isn't meteor. This should be in the meteor section.

File: package.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
 // package metadata file for Meteor.js
 
 Package.describe({
-  name: 'underscorestring:underscore.string',
-  summary: 'underscore.string (official): String manipulation extensions for Underscore.js javascript library.',
-  version: '3.0.3',
-  git: 'https://github.com/epeli/underscore.string.git',
+  name: 'taskputty:underscore.string',
```
Comment (stoeffel) [inline package.js]: shouldn't this point to our account?

**Comment by graemian** [comment]
Sorry, the changes to package.js and README.markdown (the last of the 3 commits) are not intended to be pulled. They are for my own Atmosphere package (https://atmospherejs.com/taskputty/underscore.string), which I can remove when yours is fixed.

Can you accept just the first 2 commits somehow?

**Comment by stoeffel** [comment]
@splendido Since you initially added meteor integration, do you have any thoughts on this? I have no experience with meteor. Thanks.

@graemian NP, yes I can do that.

**Comment by splendido** [comment]
I've just checked this and yes, it's my fault actually.
To *import* files, Meteor runs the code inside a closure and then picks up selected local variables to be exported: to get around this I left the global variable `s` untouched.
This seems to work if you include the package directly in your meteor project but fails if you use it from inside another package.

The solution to get around your browserify build is the one @graemian is proposing now: having a `meteor-pre.js` and a `meteor-post.js` file to eventually pick up the variable `s` to be exported within the Meteor namespace form `module.exports`.

This commit https://github.com/taskputty/underscore.string/commit/56984a153b640ad885557131a767e2236e2d2a72 should be enough to fix this issue.

In cse you're not confortable with adding two more files for the meteor integration, we might rever #405 and publish updates via the webhook trigger you're sending to [autopublish.meteor.com](http://autopublish.meteor.com/) from a forked repo keeping all Meteor files there.

Luca

**Comment by stoeffel** [comment]
> 've just checked this and yes, it's my fault actually.

NP, thanks for checking this PR.

> In cse you're not confortable with adding two more files for the meteor integration

for me it's okay. @epeli you okay with this too?

I will cherry pick the two commits from this PR later today or maybe tomorrow.

Thanks for your reply @splendido

**Pull request closed without merging**